diff -urN gnome-disk-utility-2.91.6/src/gdu-gtk/Makefile.am gnome-disk-utility-2.91.6.new//src/gdu-gtk/Makefile.am
--- gnome-disk-utility-2.91.6/src/gdu-gtk/Makefile.am	2011-01-21 20:02:39.000000000 +0000
+++ gnome-disk-utility-2.91.6.new//src/gdu-gtk/Makefile.am	2011-02-04 17:05:02.079999486 +0000
@@ -128,6 +128,7 @@
 	$(INTLLIBS)					\
 	$(LIBATASMART_LIBS)				\
 	$(AVAHI_UI_LIBS)				\
+	$(top_builddir)/src/gdu/libgdu.la		\
 	$(NULL)
 
 libgdu_gtk_la_LDFLAGS = -version-info $(LT_CURRENT):$(LT_REVISION):$(LT_AGE) \
