diff -urN gnome-disk-utility-2.91.6/src/nautilus-extension/Makefile.am gnome-disk-utility-2.91.6.new//src/nautilus-extension/Makefile.am
--- gnome-disk-utility-2.91.6/src/nautilus-extension/Makefile.am	2011-01-21 18:34:30.000000000 +0000
+++ gnome-disk-utility-2.91.6.new//src/nautilus-extension/Makefile.am	2011-02-27 11:59:00.900001406 +0000
@@ -34,7 +34,7 @@
 	$(NULL)
 
 
-nautilus_extensiondir = $(libdir)/nautilus/extensions-2.0
+nautilus_extensiondir = $(libdir)/nautilus/extensions-3.0
 nautilus_extension_LTLIBRARIES = libnautilus-gdu.la
 
 libnautilus_gdu_la_SOURCES =			\
